Makefile

```diff
@@ -5,7 +5,7 @@ GOFLAGS         :=
 BINDIR          := $(CURDIR)/bin
 GO_FILES        := $(shell find . -type d -name '.cache' -prune -o -type f -name '*.go' -print)
 GOPATH          ?= $$(go env GOPATH)
-GOPROXY         ?= $$(go env GOPROXY)
+GOPROXY         ?= $(echo $GOPROXY|awk -F ',' '{print $1}')
```

Comment:
I still believe omitting to set the `GOPROXY` environment variable in the Docker container, unless  it has been set explicitly by the user, is better. With this change, you end-up with a non-default `GOPROXY` value in the Docker container when `GOPROXY` was not set by the user, which I find surprising.

Reply:
Thanks, I removed the GOPROXY var in Makefile.